internal/pkg/cli/env_init.go

```diff
@@ -341,7 +414,17 @@ func BuildEnvInitCmd() *cobra.Command {
   /code $ copilot env init --name test --profile default
 
   Creates a prod-iad environment using your "prod-admin" AWS profile.
-  /code $ copilot env init --name prod-iad --profile prod-admin --prod`,
+  /code $ copilot env init --name prod-iad --profile prod-admin --prod
```

Comment:
Do we need `--use-default` now?

Reply:
not yet since UI prompting is not added.